### ISSUE TITLE
no way to link directly to a talk

### ISSUE THREAD

**Issue by autonome** [ipfs-shipyard/ipfs-thing-2023#170]
Speakers would like to link to their specific talk.

Internal link which opens the track pane and scrolls to the talk would be fine way to do this.

**Comment by tbenbow** [comment]
@autonome - If you were a speaker how would you like to get the link to your talk? I was thinking these links might look like...

https://2023.ipfs-thing.io/#track=track-name&talk=talk-name

The easiest thing I can think of is to add a share icon next to tracks like so...

![share-track-in-talk](https://user-images.githubusercontent.com/1904178/235492460-8f0c9ff9-5be6-4e0f-87cc-0ad0c503e160.jpg)

It's fairly subtle but maybe that is ok?

**Comment by tbenbow** [comment]
@autonome I've added this ability, I made the title links instead of using the share icon because the wrapping was sometimes strange. Here is the process to share a link to a timeslot (aka talk).

1. Open a track modal and then click a timeslot title.
2. The timeslot will be highlighted, copy the url
3. Open a new window and paste the url. The modal will open and scroll down to the timeslot with it highlighted.

PR #179
